### PR TITLE
updated plugin docs to include REX-Ray

### PR DESCRIPTION
**Pull request by clintkitson**
Adds REX-Ray as another possible volume plugin written in Go.

Signed-off-by: Clinton Kitson clintonskitson@gmail.com

### CODE REVIEW

File: experimental/plugins.md [hunk 1]
```diff
@@ -31,9 +31,14 @@ The following plugins exist:
   driver plugin which provides a virtual, multi-host network for containers.
 
 * The [Calico plugin](https://github.com/metaswitch/calico-docker) is a network
-  driver plugin which provides a multi-host network for containers with routes 
+  driver plugin which provides a multi-host network for containers with routes
   distributed by BGP.
 
+* The [REX-Ray plugin](https://github.com/emccode/rexraycli) is a volume plugin
+  which is written in Go and provides advanced storage functionality for many
+  platforms including EC2, OpenStack, XtremIO, and ScaleIO.
```
Comment (SvenDowideit) [inline experimental/plugins.md]: I don't see anything in the documentation of that repo that suggests that this is in fact a Docker volume plugin - ie, something that listens on a socket and responds to the Docker volume API

@calavera perhaps you can help me out?

**Comment by clintonskitson** [comment]
@SvenDowideit I just updated the README to indicate this further.

**Comment by calavera** [comment]
@clintonskitson we've moved the docs to `docs/extend/`, sorry about that.

**Comment by clintonskitson** [comment]
@SvenDowideit updated with latest path of docs.

**Comment by calavera** [comment]
LGMT, for what is worth :smile_cat:

**Comment by thaJeztah** [comment]
Checked the link and the linked repo does describe how to use it as a docker volumes-plugin :+1:

LGTM thanks! (and interesting as a plugin :-))

ping @fredlf @jamtur01 @moxiegirl @SvenDowideit

**Comment by thaJeztah** [comment]
Merging, @calavera's LGTM should be worth enough :wink:

**Pull request merged**